source/playgrounds.txt

```diff
@@ -97,14 +97,14 @@ specify a connection string for the deployment you want to run your
 playground against.
 
 If you have a playground file open in |vscode-short| and do not have
-an active connection, |vsce| displays
-:guilabel:`Click here to add connection` at the top of your playground.
+an active connection, |vsce| displays a
+:guilabel:`Click here to add connection` link at the top of your playground.
```

Comment:
Thanks for doing this update Max, my thoughts are:

- Link is an OK replacement for Button because it definitely looks more like a link. My only concern is that users might expect a "link" to take them to another page instead of do something within VS Code.

- On the above lines 100-101, I think "..., displays :guilabel:... at the top of your playground" was actually more clear before, without the "a...link" copy, only because the UI is very clear already and the word "link" might add that uncertainty about being linked to another page.

- Thanks for updating the position of the Run Selected Lines link and overall I think these changes are good.

Reply:
- I see how `link` might be confusing in the sense that it might remind someone of a _web_ link. However, I still feel it's less confusing than `button`.
- I undid the other change on lines 100-101.